### ISSUE TITLE
对panoptic-deeplab进行训练时，遇到了Error: /paddle/paddle/fluid/operators/softmax_with_cross_entropy_op.cu:481

### ISSUE THREAD

**Issue by ding-Chen** [PaddlePaddle/PaddleSeg#1287]
请问，在训练panoptic-deeplab时，出现了以下错误：
‘’‘
Error: /paddle/paddle/fluid/operators/softmax_with_cross_entropy_op.cu:481 Assertion `labels_[idx_lbl] >= 0 && labels_[idx_lbl] < d_ || labels_[idx_lbl] == ignore_idx_` failed. The value of label[267] expected >= 0 and < 19, or == 255,but got 24. Please check input value.
Error: /paddle/paddle/fluid/operators/softmax_with_cross_entropy_op.cu:481 Assertion `labels_[idx_lbl] >= 0 && labels_[idx_lbl] < d_ || labels_[idx_lbl] == ignore_idx_` failed. The value of label[267] expected >= 0 and < 19, or == 255,but got 24. Please check input value.
Error: /paddle/paddle/fluid/operators/softmax_with_cross_entropy_op.cu:481 Assertion `labels_[idx_lbl] >= 0 && labels_[idx_lbl] < d_ || labels_[idx_lbl] == ignore_idx_` failed. The value of label[267] expected >= 0 and < 19, or == 255,but got 24. Please check input value.
’‘’

**Comment by ding-Chen** [comment]
详细配置参照github默认
‘’‘
batch_size: 8
iters: 90000
learning_rate:
  decay:
    end_lr: 0.0
    power: 0.9
    type: poly
  value: 5.0e-05
loss:
  coef:
  - 1
  - 200
  - 0.001
  types:
  - ignore_index: 255
    top_k_percent_pixels: 0.2
    type: CrossEntropyLoss
  - ignore_index: 255
    reduction: none
    type: MSELoss
  - ignore_index: 255
    reduction: none
    type: L1Loss
model:
  align_corners: true
  aspp_out_channels: 256
  aspp_ratios:
  - 1
  - 3
  - 6
  - 9
  backbone:
    output_stride: 32
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/resnet50_vd_ssld_v2.tar.gz
    type: ResNet50_vd
  backbone_indices:
  - 2
  - 1
  - 0
  - 3
  decoder_channels: 256
  instance_aspp_out_channels: 256
  instance_class_key:
  - center
  - offset
  instance_decoder_channels: 128
  instance_head_channels: 32
  instance_low_level_channels_projects:
  - 64
  - 32
  - 16
  instance_num_classes:
  - 1
  - 2
  low_level_channels_projects:
  - 128
  - 64
  - 32
  type: PanopticDeepLab
optimizer:
  type: adam
train_dataset:
  dataset_root: data/cityscapes
  ignore_stuff_in_offset: true
  mode: train
  small_instance_area: 4096
  small_instance_weight: 3
  transforms:
  - max_scale_factor: 2.0
    min_scale_factor: 0.5
    scale_step_size: 0.25
    type: ResizeStepScaling
  - crop_size:
    - 1025
    - 513
    label_padding_value:
    - 0
    - 0
    - 0
    type: RandomPaddingCrop
  - type: RandomHorizontalFlip
  - brightness_range: 0.4
    contrast_range: 0.4
    saturation_range: 0.4
    type: RandomDistort
  - type: Normalize
  type: CityscapesPanoptic
val_dataset:
  dataset_root: data/cityscapes
  ignore_stuff_in_offset: true
  mode: val
  small_instance_area: 4096
  small_instance_weight: 3
  transforms:
  - label_padding_value:
    - 0
    - 0
    - 0
    target_size:
    - 2049
    - 1025
    type: Padding
  - type: Normalize
  type: CityscapesPanoptic
‘’‘

**Comment by ding-Chen** [comment]
PaddlePaddle版本：2.0.2
请问是哪出现问题的呢

**Comment by wuyefeilin** [comment]
类别标签不对哈，需要按照如下教程对cityscapes进行处理
https://github.com/PaddlePaddle/PaddleSeg/tree/release/2.2/contrib/PanopticDeepLab#cityscapes-1

**Comment by ding-Chen** [comment]
您好，参照您的建议程序开始运行了，不过我还有一个小问题，当我使用--do_eval     @@@@的时候，output并没有产生best_model文件夹，这是什么原因呢？@wuyefeilin
![image](https://user-images.githubusercontent.com/85236372/131202524-1f657c36-d949-45d3-af00-86a58902059b.png)

**Comment by ding-Chen** [comment]
您好，参照您的建议，程序开始运行了，我还有一个小问题，当我使用do_eval的时候，output并没有产生best_model文件夹，这是因为什么呢？ @wuyefeilin 
![微信截图_20210828095521](https://user-images.githubusercontent.com/85236372/131202619-21100f0b-8cf8-43e8-a6a9-53fad046b2e6.png)
